src/effects/Texture.tsx

```diff
@@ -20,5 +22,12 @@ export const Texture = forwardRef<TextureEffect, TextureProps>(function Texture(
     t.wrapS = t.wrapT = RepeatWrapping
   }, [t])
   const effect = useMemo(() => new TextureEffect({ ...props, texture: t || texture }), [props, t, texture])
+  const effect = useMemo(() => {
+    let tEffect = new TextureEffect({ ...props, texture: t || texture });
+    if (typeof opacity === "number") {
+      tEffect.blendMode.opacity.value = opacity;
+    }
+    return tEffect;
+  }, [props, t, texture, opacity])
```

Comment:
I'd prefer setting this as a prop instead so we don't recreate the effect and its resources if it changes (e.g. `<primitive object={effect} blendMode-opacity-value={opacity ?? 1} />`).

Reply:
Aha, I didn't know about such possibility. I'd implement that way and commit it then